rocky/account/models.py

```diff
@@ -107,7 +107,7 @@ def organizations(self) -> List[Organization]:
             return self.all_organizations
         return [
             m.organization
-            for m in filter(lambda o: o.status is not OrganizationMember.STATUSES.BLOCKED, self.organization_members)
+            for m in filter(lambda o: o.status is not OrganizationMember.blocked, self.organization_members)
```

Comment:
This shouldn't check `o.status` but just `o.blocked`

Reply:
Good catch, fixed it 👍